README.md

```diff
@@ -30,9 +30,29 @@ docker run -e EMAIL=<EMAIL> -e PASSWORD=<PASSWORD> epicgames
 Replacing the environment variables `EMAIL` and `PASSWORD` for your epicgames store credentials.
 
 ## Optional configuration
-You can also specify two extra environmente variables `TIMEOUT` and `LOGIN_TIMEOUT` if you have a slow internet connection speed and want to make sure it won't affect the result of the script. `LOGLEVEL` can be used to specify the [log level](https://docs.python.org/3.7/library/logging.html#logging-levels) to log.
+You can also specify two extra environmente variables `TIMEOUT` and `LOGIN_TIMEOUT` if you have a slow internet connection speed and want to make sure it won't affect the result of the script. `LOGLEVEL` can be used to specify the [log level](https://docs.python.org/3.7/library/logging.html#logging-levels) to log. `SLEEPTIME` can be used to set the number of seconds to wait between looping through the procedure again. Defaults to -1, which will stop execution after a single iteration.
 Run a docker container with these:
 ```
-docker run -e TIMEOUT=10 -e LOGIN_TIMEOUT=15 -e LOGLEVEL=DEBUG -e EMAIL=<EMAIL> -e PASSWORD=<PASSWORD> epicgames
+docker run -e TIMEOUT=10 -e LOGIN_TIMEOUT=15 -e LOGLEVEL=DEBUG -e SLEEPTIME=43200 -e EMAIL=<EMAIL> -e PASSWORD=<PASSWORD> epicgames
 ```
 These environmente variables have a default value of `TIMEOUT = 5` and `LOGIN_TIMEOUT = 10`.
+
+## Docker Compose
+```
+version: '2'
+
+services:
+
+    egs-freegame:
+        build:
+            context: ./epicgames-weekly-freegames/
```

Comment:
This means you will create the docker-compose file outside of the project dir?

Reply:
Yes. Which is how I've always done it. Since I'm going to provide a docker compose file within the project as well, that one will assume that it is placed inside the project directory.

Ideally the project image would be available on Docker Hub. Is that something you'd be willing to do? Then that entire build block would be replaced with `image: ricardo-osorio/epicgames-weekly-freegames`, or something similar.

Reply:
Yah I will look into that!